components/PictureGrid.js

```diff
@@ -1,20 +1,24 @@
 import React, { PropTypes } from 'react';
+import Picture from '../components/Picture';
 
-const PictureGrid = () => {
-  const pictures = [{
-    url: 'blah',
-  }];
-
+const PictureGrid = (props) => {
+  const pictureItems = props.pictures.map((picture) =>
+    <Picture
+      key={picture.name}
+      url={picture.url}
+      name={picture.name}
+      title={picture.title}
+    />,
+  );
   return (
-    <div className="PictureGrid">
-      PictureGrid!
-      {pictures[0].url}
+    <div className="pictureGrid">
+      {pictureItems}
```

Comment:
nit: you can just put the entire map inside `{}` instead of setting it to pictureItems
```js
<div>
  {pictures.map((picture) => (
    <Picture />
  )}
</div>
```